SWIPs/swip-honey_to_money.md

```diff
@@ -42,10 +42,10 @@ In the current implementation, honey prices are hardcoded. When this price in co
 
 ## Rationale
 We need a way for prices in the Swarm to change over time. However, due to the nature of Swarm - specifically independent connections between pairs of nodes each with their individual accounting and payments - there is no clear price discovery mechanism. Furthermore, since Swarm nodes are expected to be conduits for both payments and data (buy from one peer and sell to the other) it is necessary for nodes to coordinate price changes with all their peers. In short, we need a mechanism for network-wide changes in pricing that the peer-to-peer routing and accounting does not provide.
```

Comment:
"since Swarm nodes are expected to be conduits for both payments and data (buy from one peer and sell to the other)"

I think this sentence is confusing, what does it even mean "to be conduits". Are you meaning to say that it works bothways? Sell and Buy? Then just say that please

Reply:
@homotopycolimit

Reply:
No I mean they are conduits for payments and data.
A wants data, Z has the data. A requests data from B who requests it from C who requests it ... from Z. 
Y pays Z. X pays Y..... A pays B.
The end result is that data has moved from Z to A and net payment has moved from A to Z. The other nodes were just conduits for the data and the payment. They are able to cache the data in order to profit from it in future, but for this particular exchange they are conduits. "B buys from C but sells to A"

Reply:
I still think the sentence is confusing, why "being conduits for both payments and data" makes it necessary to coordinate price changes with all their peers?

Isn't the gist that "we need a mechanism for network-wide changes"? Can't we just say that?